DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -108,6 +114,7 @@ FROM Centres
                             c.pwTelephone AS CentreTelephone,
                             c.pwEmail AS CentreEmail,
                             c.pwPostCode AS CentrePostcode,
+                            c.ShowOnMap AS ShowCentreOnMap,

```

Comment:
I'd be happy for this to be simply "ShowOnMap" given that it lives on a data object.